src/Razor/Microsoft.NET.Sdk.Razor/test/Microsoft.NET.Sdk.Razor.Test.csproj

```diff
@@ -13,4 +13,13 @@
     <Reference Include="Microsoft.NET.Sdk.Razor" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="TestFiles\Generated\*.rz.scp.css">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="TestFiles\Generated\*.bundle.scp.css">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
```

Comment:
Is this intentional? Or VS weirdness?

Reply:
Intentional, these need to be copied to the output folder for the unit tests to pick them up (They are part of the concat tests)

Reply:
OK thanks